### PR TITLE
implementation method matchResult() in class FootballMatch

### PR DESCRIPTION
**Pull request by KrohIgor**

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1 && result2 == bet2) ? 2 :
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]: '?' should be on a new line

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
...
+                (result1 > result2 && bet1 > bet2)
+                        || (result1 < result2 && bet1 < bet2)
+                        || (result1 == result2 && bet1 == bet2) ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
                (result1 > result2 && bet1 > bet2)
             || (result1 < result2 && bet1 < bet2)
             || (result1 == result2 && bet1 == bet2) 
             ? 1 : 0;
```

**Review by dmytro-hryhoruk — COMMENTED** [review]
remember about check style https://mate-academy.github.io/style-guides/java/java.html

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 1 above)
Comment (KrohIgor) [reply src/main/java/core/basesyntax/FootballMatch.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**